packages/shared/src/hooks/useUserMention.ts

```diff
@@ -79,7 +81,7 @@ export function useUserMention({
         requestMethod(
           `${apiUrl}/graphql`,
           RECOMMEND_MENTIONS_QUERY,
-          { postId, query },
+          { postId, query, sourceId },
```

Comment:
I'm not sure if we need to invalidate the "old" key manually maybe?

Reply:
It is a good practice to keep the parameter of the query inside the key itself. This is to ensure cache is accurate as it can get.

Reply:
I know, but you changed a existing query key.
Wondering if it has any impact on the existing apps cache?